sunpy/net/dataretriever/sources/noaa.py

```diff
@@ -157,7 +154,7 @@ class SRSClient(GenericClient):
     <BLANKLINE>
 
     """
-    BASE_URL = 'ftp://ftp.swpc.noaa.gov/pub/warehouse/'
+    BASE_URL = 'ftp://ftp.ngdc.noaa.gov/STP/swpc_products/daily_reports/solar_region_summaries/'
```

Comment:
I can't get parfive handle this url via aioftp. I am doing something wrong, I just have not been able to figure out what.